Sources/Testing/SourceAttribution/SourceLocation.swift

```diff
@@ -74,20 +78,48 @@ public struct SourceLocation: Sendable {
   public var _filePath: String
 
   /// The line in the source file.
+  ///
+  /// - Precondition: The value of this property must be greater than or equal
```

Comment:
You could rewrite this as just "greater than `0`".

Reply:
Sure. I had adapted the phrasing from a doc comment I saw you had written for `Configuration.maximumIterationCount` 😜